docs/src/pages/[platform]/components/menu/react.mdx

```diff
@@ -152,11 +182,13 @@ import '@aws-amplify/ui-react/styles.css';
 
 import './styles.css';
 
-<Menu className="my-menu-content" triggerClassName="my-menu-trigger">
-  <MenuItem>Download</MenuItem>
-  <MenuItem>Create a Copy</MenuItem>
-  <MenuItem>Mark as Draft</MenuItem>
-</Menu>;
+export const ClassNameExample = () => (
```

Comment:
Shouldn't these move to example files and get imported from there? Don't know if we are doing that for styling examples.